CHANGELOG.md

```diff
@@ -16,7 +16,10 @@ Possible log types:
 - [added] The `Sensors`, `PeopleNowPresentSensor` and `TemperatureSensor`
   structs now derive `Default` (#84)
 - [added] Basic support for the new v14 API (#85) This is a breaking change since
-  it changes the `api` field of `Status` to `Option<String>`.
+  it changes the `api` field of `Status` to `Option<String>`. Includes
+  * Rename jabber to xmpp in contact field
```

Comment:
```suggestion
  * Add `xmpp` field in `Contact` struct (the `jabber` field is not accepted anymore when targeting v14)
```

Reply:
Hmm maybe a bit shorter like: "Add `xmpp` field in `Contact` struct and deprecate the `jabber` field"?

Reply:
👍🏻

Reply:
Fixed.

Reply:
(Thanks. The PR is approved, feel free to merge, no need to wait for me 🙂)